.github/workflows/aws-deploy.yaml

```diff
@@ -17,43 +17,57 @@ jobs:
     needs: pre-commit
     if: github.ref == 'refs/heads/main'
     steps:
+
       - name: Checkout repository
         uses: actions/checkout@v2
+
       - name: Create directory for remote sceptre templates
-        run:  mkdir -p templates/remote/
-      - name: Assume AWS Role
+        run: mkdir -p templates/remote/
+
+      - name: Set up python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Set up dependency caching
+        uses: actions/cache@v2
+        with:
+          path: ~/.local/share/virtualenvs
+          key: ${{ runner.os }}-pipenv-${{ hashFiles('Pipfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pipenv-
+
+      - name: Install dependencies
+        run: |
+          python -m pip install pipenv
+          pipenv install
```

Comment:
shouldn't this be `pipenv install --dev`?

Reply:
see above